source/generate_cids/generate_concept_dicts.py

```diff
@@ -210,67 +210,6 @@ def get_table_d1():
 
 
 def write_concepts(concepts, cid_concepts, cid_lists, name_for_cid):
-    lines = (DOC_LINES +
```

Comment:
Better use this version, as it uses `DOC_LINES` for the duplicated part.

Reply:
Ah, that's right. Fixed now.